.storybook/preview.jsx

```diff
@@ -10,7 +10,7 @@ export const parameters = {
   actions: { argTypesRegex: '^on.*' },
 };
 
-const withChakra = (StoryFn: Function, context: StoryContext) => {
+const withChakra = (StoryFn, _context) => {
```

Comment:
Does it hurt to keep the typings in here?

Reply:
Yea because I had to switch it to js for storybook to build. They have some kind of bug in their build script with typescript files right now I guess